content/teams_intro.tex

```diff
@@ -80,9 +80,9 @@ \subsubsection*{Team Creation}
 This configuration argument is of type \CTYPE{shmem\_team\_config\_t}, which
 is detailed further in Section~\ref{subsec:shmem_team_config_t}.
 
-\acp{PE} in a newly created teams are consecutively numbered with starting with
-\ac{PE} number 0. \acp{PE} are always ordered by the existing global \ac{PE} number that
-would be returned by the \FUNC{shmem\_my\_pe} routine. Team relative \ac{PE}
+\acp{PE} in a newly created teams are consecutively numbered starting with
+\ac{PE} number 0. \acp{PE} are ordered by their \ac{PE} number in
+the parent team. Team relative \ac{PE}
```

Comment:
For teams that don't have a parent, how do you determine the order? Do we need to specify that or is it redundant?

Reply:
The only teams that "don't have a parent" are `SHMEM_TEAM_WORLD` and `SHMEM_TEAM_SHARED` (cf. #306). Since this is talking about "newly created teams", this (presumably) means teams created by a call to `shmem_team_split_*`. By construction, they all are split from a parent team.